tests/TimeTest.php

```diff
@@ -112,4 +117,15 @@ public function it_should_not_mutate_passed_datetime()
         $this->assertEquals('2016-09-27 15:00:00', $time->toDateTime($dateTime)->format('Y-m-d H:i:s'));
         $this->assertEquals('2016-09-27 12:00:00', $dateTime->format('Y-m-d H:i:s'));
     }
+
+    /** @test */
+    public function it_should_not_mutate_passed_datetime_immutable()
+    {
+        $this->markTestIncomplete('Not supported in v1.x');
```

Comment:
Whats the reason for adding incompatible tests?

Reply:
As explained above, it shows explicitly the ongoing support of immutability. Document known limitations is a good practice.